src/main/java/org/reaktivity/nukleus/mqtt/internal/MqttConfiguration.java

```diff
@@ -21,24 +21,33 @@
 
 public class MqttConfiguration extends Configuration
 {
-    public static final String PUBLISH_TIMEOUT_NAME = "nukleus.mqtt.publish.timeout";
-    public static final String CONNECT_TIMEOUT_NAME = "nukleus.mqtt.connect.timeout";
-    public static final String CLIENT_ID_NAME = "nukleus.mqtt.client.id";
-    public static final String RETAIN_AVAILABLE_NAME = "nukleus.mqtt.retain.available";
-
     private static final ConfigurationDef MQTT_CONFIG;
     public static final LongPropertyDef CONNECT_TIMEOUT;
     public static final LongPropertyDef PUBLISH_TIMEOUT;
     public static final PropertyDef<String> CLIENT_ID;
+    public static final IntPropertyDef SESSION_EXPIRY_INTERVAL;
+    public static final BytePropertyDef MAXIMUM_QOS;
     public static final BooleanPropertyDef RETAIN_AVAILABLE;
+    public static final ShortPropertyDef TOPIC_ALIAS_MAXIMUM;
+    public static final BooleanPropertyDef WILDCARD_SUBSCRIPTION_AVAILABLE;
+    public static final BooleanPropertyDef SUBSCRIPTION_IDENTIFIERS_AVAILABLE;
+    public static final BooleanPropertyDef SHARED_SUBSCRIPTION_AVAILABLE;
+    public static final BooleanPropertyDef NO_LOCAL;
 
     static
     {
         final ConfigurationDef config = new ConfigurationDef("nukleus.mqtt");
         PUBLISH_TIMEOUT = config.property("publish.timeout", TimeUnit.SECONDS.toSeconds(30));
         CONNECT_TIMEOUT = config.property("connect.timeout", TimeUnit.SECONDS.toSeconds(3));
         CLIENT_ID = config.property("client.id", "client");
+        SESSION_EXPIRY_INTERVAL = config.property("session.expiry.interval", 0);
+        MAXIMUM_QOS = config.property("maximum.qos", (byte) 0);
         RETAIN_AVAILABLE = config.property("retain.available", true);
+        TOPIC_ALIAS_MAXIMUM = config.property("topic.alias.maximum", (short) -1);
```

Comment:
If we don't support `topic alias` then this should default to `0`, if we do then it should default to the size we support by default rather than `-1`.